docs/root/version_history/current.rst

```diff
@@ -28,6 +28,7 @@ Removed Config or Runtime
 
 New Features
 ------------
+* stats: allow configuring histogram buckets for prometheus or metrics_service.
```

Comment:
I would drop the "for prometheus..". Presumably this applies to any stat sink (including proprietary) that uses histograms.

Reply:
Clarified